pages/common/gh.md

```diff
@@ -0,0 +1,32 @@
+# gh
+
+> Work seamlessly with GitHub from the command line.
+> More information: <https://cli.github.com/>.
+
+- Create a pull request:
+
+`gh pr create`
+
+- View a pull request in the browser:
+
+`gh pr view`
```

Comment:
This one is missing argument as well. It should be noted which PR is to be viewed.